mmv1/templates/terraform/examples/sql_sqlserver_instance_backup.tf.erb

```diff
@@ -0,0 +1,15 @@
+# [START cloud_sql_sqlserver_instance_backup]
+resource "google_sql_database_instance" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
tests need 
```
  deletion_protection =  "false"
```
so we can delete the resource after

Reply:
done